variable-manipulation.js

```diff
@@ -0,0 +1,23 @@
+const TAX_RATE = 1.1
+const SHIPPING_DEFAULT = 5
+
+function calculateTotal(items, options = {}) {
+  if (items == null || items.length === 0) return 0
+
+  let total = 0
+  items.forEach(item => {
+    total += item.price * item.quantity
+  })
+  total = total - total * (options.discount || 0)
+  total = total * TAX_RATE
+  if (options.shipping !== 0) {
```

Comment:
**GPT COMMENT**: 
 - The condition on line 13 checks if options.shipping is not equal to 0, but it doesn't handle the case when options.shipping is undefined. Consider adding a condition to handle this case.